apps/examples/testcase/Makefile

```diff
@@ -195,8 +195,9 @@ ifeq ($(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_UTC),y)
 endif
 ifeq ($(CONFIG_EXAMPLES_TESTCASE_TASK_MANAGER_ITC),y)
 	$(Q) $(call REGISTER,taskmgr_itc,itc_taskmanager_main,TASH_EXECMD_ASYNC,100,2048)
+	$(Q) $(call REGISTER,tm_builtin,tm_builtin_main,TASH_EXECMD_ASYNC,101,1024)
 	$(Q) $(call REGISTER,tm_itc,tm_itc_main,TASH_EXECMD_ASYNC,101,1024)
-	$(Q) $(call REGISTER,itc_taskmanager_user1,itc_taskmanager_user1_main,TASH_EXECMD_ASYNC,101,1024)
+	$(Q) $(call REGISTER,taskmanager_user1,taskmanager_user1_main,TASH_EXECMD_ASYNC,101,1024)
```

Comment:
What is user1 test (command)? I understand others, tm_broadcast is for testing broadcast functionality, tm_utc for utc test, tm_itc for itc. But I don't understand what user1 is. Please rewrite the command name. function name also.

Reply:
changed it